File: JavaScript solutions/448. Find All Numbers Disappeared in an Array.js
/**
 * @param {number[]} nums
 * @return {number[]}
 */
var findDisappearedNumbers = function (nums) {
  let len = nums.length;
  const v = [],
    ans = [];
  for (let i = 0; i <= len; i++) v.push(0);

  for (let i = 0; i < len; i++) v[nums[i]] = 1;

  for (let i = 1; i <= len; i++) {
    if (v[i] == 0) ans.push(i);
  }
  return ans;
};

File: C++ solutions/448. Find All Numbers Disappeared in an Array.cpp
class Solution {
public:
    vector<int> findDisappearedNumbers(vector<int>& nums) {
        int len=nums.size();
        vector<int>v,ans;
        for(int i=0;i<=len;i++) v.push_back(0);

        for(int i=0;i<len;i++) v[nums[i]]=1;

         for(int i=1;i<=len;i++){
             if(v[i]==0) ans.push_back(i);
         }
         return ans;
    }
};